schemes/article/footer.js

```diff
@@ -0,0 +1,16 @@
+export default {
+  props: {
+    likes: {
+      type: Array,
+      default: () => []
+    },
+    comments: {
```

Comment:
Likes and comments should be a number, not an array. I'm not sure but, every article has commentsCount property.